data/hai/hai outfits.txt

```diff
@@ -118,8 +118,8 @@ outfit "Ion Cannon"
 		"hit force" 120
 		"shield damage" 168
 		"hull damage" 60
-		"ion damage" 5
-		"scrambling damage" 5
+		"ion damage" 6.5
+		"scrambling damage" 4.5
```

Comment:
```suggestion
		"ion damage" 6
		"scrambling damage" 4.5
```
just opening this to get a place of discussion for the values
I'd be content with 6 ion too if ppl feel like this is too much of a buff

Reply:
it's not supposed to be too good, just not suck
I agree it may not be idea to split them too much and leave that for the prototypes, but 6.5 and 6 is pushing it
```suggestion
		"ion damage" 6
		"scrambling damage" 6
```
let's let ppl just upvote the suggestion they like most or just suggest one themselves

Reply:
These suggestions all seem ok from the balance side. I personally prefer 6.5 ion and 4.5 scrambling, but the point about not wanting the two to be significantly different is a fair point. I'd like to get more opinions on that, but I think either 6.5/4.5 or 6/6 would be good.